code/modules/vore/eating/simple_animal_vr.dm

```diff
@@ -34,20 +34,19 @@
 	var/mob/living/carbon/human/user = usr
 	if(!istype(user) || user.stat) return
 
-	var/datum/belly/B = vore_organs[vore_selected]
 	if(retaliate || (hostile && faction != user.faction))
 		user << "<span class='warning'>This predator isn't friendly, and doesn't give a shit about your opinions of it digesting you.</span>"
 		return
-	if(B.digest_mode == "Hold")
+	if(vore_selected.digest_mode == "Hold")
```

Comment:
I recommend switching to DM_* constants

Reply:
Didn't touch that, but I suppose this is a refactor, so fixed.